terraform/modules/validators/variables.tf

```diff
@@ -533,3 +533,12 @@ variable "tf_script_version" {
   type        = string
   description = "The version of the provisioning scripts located in the OCI image used to create the WebLogic compute instances"
 }
+
+variable "tenancy_id" {
```

Comment:
@roberto-sanchez-herrera @Mahuwa-Barman
Can you review this? I'd like to have consensus that adding tenancy_id as a variable is the way to approach this or if there is a data source way that should be looked into for getting this value.

Reply:
Nevermind. I missed that this variable is already defined at the main level.